src/devrules/cli_commands/group.py

```
@@ -0,0 +1,640 @@
+            typer.secho("Group name is required.", fg="red")
+            raise typer.Exit(1)
+
+        config_path = find_config_file()
+        if not config_path:
+            typer.secho("Configuration file not found", fg="red")
+            raise typer.Exit(1)
+
+        # Load raw toml to preserve comments and structure as much as possible
+        try:
+            data = toml.load(config_path)
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)